lib/active_merchant.rb

```diff
@@ -59,50 +59,13 @@ def Base64.strict_encode64(v)
 require 'timeout'
 require 'socket'
 
-require 'active_utils/common/network_connection_retries'
-silence_warnings{require 'active_utils/common/connection'}
-require 'active_utils/common/post_data'
-require 'active_utils/common/posts_data'
+require 'active_merchant/network_connection_retries'
+require 'active_merchant/connection'
+require 'active_merchant/post_data'
+require 'active_merchant/posts_data'
 
 require 'active_merchant/billing'
 require 'active_merchant/version'
 require 'active_merchant/country'
 
 I18n.enforce_available_locales = false
-
-module ActiveMerchant #:nodoc:
-  OFFSITE_PAYMENT_EXTRACTION_MESSAGE = "Integrations have been extracted into a separate gem (https://github.com/Shopify/offsite_payments) and will no longer be loaded by ActiveMerchant 2.x."
-
-  module Billing #:nodoc:
-    def self.const_missing(name)
-      if name.to_s == "Integrations"
-        ActiveMerchant.deprecated(OFFSITE_PAYMENT_EXTRACTION_MESSAGE)
-        require "active_merchant/offsite_payments_shim"
-        ActiveMerchant::OffsitePaymentsShim
-      else
-        super
-      end
-    end
-
-    def self.included(klass)
-      def klass.const_missing(name)
-        if name.to_s == "Integrations"
-          ActiveMerchant.deprecated(OFFSITE_PAYMENT_EXTRACTION_MESSAGE)
-          require "active_merchant/offsite_payments_shim"
-          ActiveMerchant::OffsitePaymentsShim
-        else
-          super
-        end
-      end
-    end
-  end
-
-  def self.deprecated(message, caller=Kernel.caller[1])
```

Comment:
Actually I think it is used elsewhere. I'll leave it for now and remove it in a separate PR.